src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectProtocolValidationContext.cs

```diff
@@ -59,8 +53,13 @@ public class OpenIdConnectProtocolValidationContext
         public string State { get; set; }
 
         /// <summary>
-        /// Gets or sets a validated id_token.
+        /// Gets or sets the response received from userinfo_endpoint.
+        /// </summary>
+        public JwtSecurityToken UserInfoEndpointResponse { get; set; }
```

Comment:
The JWT is only one field we get back from the user info endpoint, no? Which field?